config/locales/en.yml

```diff
@@ -2826,7 +2826,6 @@ See the %{link} to find out more about %{sitename}'s features and to start using
   contact: "Contact"
   web: "Web"
   primary_details: "Primary Details"
-  adrdress: "Address"
```

Comment:
It actually contains a typo. Very unlikely that it's used if you can't find the string.